rdkit/Chem/PandasTools.py

```diff
@@ -531,25 +554,47 @@ def AlignToScaffold(frame, molCol='ROMol', scaffoldCol='Murcko_SMILES'):
   frame[molCol] = frame.apply(lambda x: AlignMol(x[molCol], x[scaffoldCol]), axis=1)
 
 
-if __name__ == "__main__":
-  import sys
+# ==========================================================================================
+# Monkey patching RDkit functionality
+def InstallPandasTools():
+  """ Monkey patch a few RDkit methods of Chem.Mol """
+  global _originalSettings
+  if len(_originalSettings) == 0:
+    _originalSettings['Chem.Mol.__ge__'] = Chem.Mol.__ge__
+    _originalSettings['Chem.Mol.__str__'] = Chem.Mol.__str__
+  rdchem.Mol.__ge__ = _molge
+  rdchem.Mol.__str__ = PrintAsBase64PNGString
+
+
+def UninstallPandasTools():
+  """ Monkey patch a few RDkit methods of Chem.Mol """
+  global _originalSettings
+  Chem.Mol.__ge__ = _originalSettings['Chem.Mol.__ge__']
+  Chem.Mol.__str__ = _originalSettings['Chem.Mol.__str__']
+
+
+_originalSettings = {}
+InstallPandasTools()
```

Comment:
Here I combine all the modifications that this module makes into the InstallPandasTools function. It keeps track of the original settings and allows to remove the PandasTools modifications. This is important for testing, if the default test runner from unittest with test discovery is used. 

The change is equivalent to the modification I made to IPythonTools a while ago.